zb-property.js

```diff
@@ -430,6 +440,47 @@ class ZigbeeProperty extends Property {
     ];
   }
 
+  /**
+   * @method parseEnumAttr
+   *
+   * Converts a thermostat mode attribute into a property value (string).
+   */
+  parseEnumAttr(attrEntry) {
+    const attrData = attrEntry.attrData;
+    let propertyValue;
+
+    if (this.hasOwnProperty('enumArray') && attrData < this.enumArray.length) {
+      // We use enumArray when we want the UI to display like a string
```

Comment:
Seems silly to me to add extra things here just to get around UI issues. Are there other things we could do? Maybe if the property is `readOnly`, we don't display the `enum` as a &lt;select&gt;?

Reply:
yeah - I think that a readOnly enum should look like a string. If I use enum you get the little arrowa that suggest you can pick something but you can't.

Aside: I also think that a readOnly string should have the same background as the bubble (i.e. look something like say the battery voltage).

Reply:
I agree. Could you file an issue on the gateway and let @benfrancis weigh in?